app/src/pages/inside/logsPage/logsTabContent/logsTabContent.jsx

```diff
@@ -0,0 +1,248 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, Fragment } from 'react';
+import track from 'react-tracking';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { connectRouter } from 'common/utils';
+import {
+  logItemsSelector,
+  logPaginationSelector,
+  loadingSelector,
+  pageLoadingSelector,
+  NAMESPACE,
+  LOG_LEVEL_FILTER_KEY,
+  WITH_ATTACHMENTS_FILTER_KEY,
+  HIDE_PASSED_LOGS,
+  HIDE_EMPTY_STEPS,
+  getLogLevel,
+  setLogLevel,
+  DETAILED_LOG_VIEW,
+  logViewModeSelector,
+  LOG_STATUS_FILTER_KEY,
+  isLogPageWithNestedSteps,
+} from 'controllers/log';
+import { withFilter } from 'controllers/filter';
+import { withPagination, PAGE_KEY, DEFAULT_PAGINATION, SIZE_KEY } from 'controllers/pagination';
+import { withSortingURL, SORTING_ASC } from 'controllers/sorting';
+import { userIdSelector } from 'controllers/user';
+import { PaginationToolbar } from 'components/main/paginationToolbar';
+import { LogsGrid } from '../logsGrid';
+import { LogsGridToolbar } from '../logsGridToolbar';
+import { SauceLabsSection } from '../sauceLabsSection';
+
+@connect(
+  (state) => ({
+    logItems: logItemsSelector(state),
+    loading: loadingSelector(state),
+    pageLoading: pageLoadingSelector(state),
+    userId: userIdSelector(state),
+    logViewMode: logViewModeSelector(state),
+    isNestedStepView: isLogPageWithNestedSteps(state),
+  }),
+  null,
+)
+@withSortingURL({
+  defaultFields: ['logTime'],
+  defaultDirection: SORTING_ASC,
+  namespace: NAMESPACE,
+})
+@withFilter({
+  filterKey: 'filter.cnt.message',
+  namespace: NAMESPACE,
+})
+@withPagination({
+  paginationSelector: logPaginationSelector,
+  namespace: NAMESPACE,
+})
+@connectRouter(
+  (query) => ({
+    logLevelId: query[LOG_LEVEL_FILTER_KEY],
+    logStatus: query[LOG_STATUS_FILTER_KEY],
+    withAttachments: query[WITH_ATTACHMENTS_FILTER_KEY],
+    hideEmptySteps: query[HIDE_EMPTY_STEPS],
+    hidePassedLogs: query[HIDE_PASSED_LOGS],
+  }),
+  {
+    onChangeLogLevel: (userId, logLevel) => {
+      setLogLevel(userId, logLevel);
+      return { [LOG_LEVEL_FILTER_KEY]: logLevel.id, [PAGE_KEY]: 1 };
+    },
+    onChangeWithAttachments: (withAttachments) => {
+      return { [WITH_ATTACHMENTS_FILTER_KEY]: withAttachments || undefined };
+    },
+    onChangeHideEmptySteps: (hideEmptySteps) => {
+      return { [HIDE_EMPTY_STEPS]: hideEmptySteps || undefined };
+    },
+    onChangeHidePassedLogs: (hidePassedLogs) => {
+      return { [HIDE_PASSED_LOGS]: hidePassedLogs || undefined };
+    },
+    onChangeLogStatusFilter: (status) => ({
+      [LOG_STATUS_FILTER_KEY]: status || undefined,
+    }),
+  },
+  { namespace: NAMESPACE },
+)
+@track()
+export class LogsTabContent extends Component {
```

Comment:
This component looks similar to `LogsPage` component.
Since `LogsPage` is an extended version of `LogsTabContent`. You should use `LogsTabContent` inside of `LogsPage`.
Since this component will be responsible not only for the tab but also for filling the content on the log view page, it must be renamed, f.e. `LogsGridWrapper`